app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -408,6 +409,33 @@ def test_update_experiment_is_rollout(self):
         experiment = NimbusExperiment.objects.get(id=experiment_id)
         self.assertTrue(experiment.is_rollout)
 
+    def test_update_live_rollout(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            is_rollout=True,
+            population_percent=40,
+        )
+        experiment_id = experiment.id
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "isRollout": True,
+                    "populationPercent": "50",
+                    "publishStatus": NimbusConstants.PublishStatus.DIRTY.name,
+                    "changelogMessage": "test changelog message",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
+        experiment = NimbusExperiment.objects.get(id=experiment_id)
+        self.assertTrue(experiment.is_rollout)
+        self.assertEqual(experiment.population_percent, Decimal("50"))
```

Comment:
We're setting the experiment up in IDLE, and I'd expect to assert that it's in DIRTY after the save?